performance/rp_federation_config.py

```diff
@@ -0,0 +1,18 @@
+import json
+import os
+
+from performance import config
+
+
+class RPFederationConfig(dict):
```

Comment:
Are we sure we really want to create a new config class for every single config file we may read? And then _in each case_ test that constructing it actually reads a file? This feels like it could get a bit over-the-top quite quickly!

Perhaps these could be properties of the existing `Config` class?

Reply:
You are correct, but this class has only been lifted out of an existing module, where it shouldn't belong anymore since we're now using it to get the RP mappings from other places (i.e. `get_df_for_all_rps`).

Reply:
Hmm, ok, certainly that's fine for now. I'm not massively happy about the precedent we're setting by having that test, though - I'll think on it.